site/docs/plugins/ratelimiter.md

```diff
@@ -99,48 +103,91 @@ bot.use(
     keyGenerator: (ctx) => {
       return ctx.from?.id.toString();
```

Comment:
We do not use `return` for promises. This has no advantage, but can be confusing.
```suggestion
    keyGenerator: async (ctx) => {
      await ctx.from?.id.toString();
```

Reply:
The type of `keyGenerator` is `(ctx: C) => string | undefined`.

Reply:
I am blind, forget it.